Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.cpp

```diff
@@ -1180,6 +1180,9 @@ void NetworkConnectionToWebProcess::writeBlobsToTemporaryFilesForIndexedDB(const
         if (auto* session = networkSession())
             session->protectedStorageManager()->registerTemporaryBlobFilePaths(Ref { m_connection }, filePaths);
 
+        for (auto& path : filePaths)
+            allowAccessToFile(path);
```

Comment:
We talked about why this is needed on Slack.
I think it's worth covering in a comment.